### ISSUE TITLE
Use OVRPlayerController with interaction sdk

### ISSUE THREAD

**Issue by dhruvmodhapp** [XRBootcamp/InteractionSDK#1]
Hello,
I tried a different way to integrate OVRPlayercontroller with the new Interaction oculus SDK. But not works well.

Does anyone have an idea how to make it work?

**Comment by lucas-martinic** [comment]
Hi, did you try starting from the demo scenes in Assets > Oculus > Interaction > Samles > Scenes?

**Comment by dhruvmodhapp** [comment]
> Hi, did you try starting from the demo scenes in Assets > Oculus > Interaction > Samles > Scenes?

Yes, I tried the demo sample scene. With Simple OVRCameraRig it working. But not work with OVRPlayerController. 
I tried Different ways to use it with OVRPlayerController but no luck.
